_posts/2017/2017-02-23-scrum-interview.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "Scrum Master insights at ePages"
+date: "2017-02-09 07:00:00"
+categories: working-at-epages
+authors: ["Rose", "Ann-Katrin"]
+---
+
+As already argued in our previous [blog post](https://developer.epages.com/blog/2017/01/31/getting-twice-the-things-done-in-half-the-time.html) the agile process framework [Scrum](https://www.scrum.org/resources/what-is-scrum) is our preferred methodology in order to manage our product development at ePages.
+To get further insights into our work with Scrum, we interviewed our Scrum Master Ann-Katrin:
+
+### Ann-Katrin, please introduce yourself!
+
+Hi, I am Ann-Katrin, the Scrum Master here at ePages.
+I started my career at a small agency, which was specialized in doing e-commerce for big retailers, here in Hamburg.
+This is where I worked for about seven years, before I moved to London.
+In London I worked for two big retailers, namely [Arcadia](https://www.arcadiagroup.co.uk/) and [Debenhams](http://www.debenhams.com/).
+At the beginning, I was dedicated more towards the operational business, yet then I discovered my love for project management where I set my focus on agile projects.
+So, it was less project management, but rather working with agile teams and being the one coordinating their work.
+This is when I got to know the processes of Scrum and I fell in love right away.
+Therefore, I got certificated as a Scrum Master and ever since worked in that field.
+Back then, some of the teams I worked with were already using Scrum and for the other teams I helped them to introduce it to improve their working processes.
+
+At one point, it was time for me to move back to Germany, because I simply wanted to be closer to my friends again.
+I first worked for a small agency that was specialized in App development, which was fun, yet I really missed working with bigger teams.
+I normally only had to coordinate a team that consisted out of one UI designer and two developers for one app project.
+
+Ever since I started at ePages, I am very thrilled to be working with bigger teams as well as interesting and passionate people.
+
+### What are the main differences between agile and traditional project management?
+
+In traditional project management you have got your fixed tasks that you do one after another.
+For example, you start with a big conception phase, then you go on with a design phase as well as development and eventually at the end the customer would see the product.
+However, until then it remains completely non-transparent for the customer on what exactly the development team is working on.
+Only at the end, he will see the product and might realize that he has changed his mind and demands some product changes.
```

Comment:
Only at the end, they will see... and might realize that they changed their mind and demand...